### ISSUE TITLE
Add temperature and length

### ISSUE THREAD

**Issue by liam-maps** [fieldOfView/Cura-CustomJobPrefix#32]
Hi!
I'm finding this plugin very handy, but I'm missing having those two values on my filenames.
Could it be added in future releases?
Thank you very much.

**Comment by armin771** [comment]
Would also be very happy if I could add the temperature of the hotend and bed.

**Comment by tstone52** [comment]
I agree. This plugin is really great, but would be even better if the hotend temperature could be added to filenames. I always include temp in my filenames and to have it be automatic would be awesome! =)

**Comment by mooklepticon** [comment]
Another vote for temperature.
